pkg/dashboards/thanos/thanos_query.go

```diff
@@ -0,0 +1,80 @@
+package thanos
+
+import (
+	"github.com/perses/perses/go-sdk/dashboard"
+	panelgroup "github.com/perses/perses/go-sdk/panel-group"
+	labelValuesVar "github.com/perses/perses/go-sdk/prometheus/variable/label-values"
+	listVar "github.com/perses/perses/go-sdk/variable/list-variable"
+
+	"github.com/perses/community-dashboards/pkg/dashboards"
+	panels "github.com/perses/community-dashboards/pkg/panels/thanos"
+	"github.com/perses/community-dashboards/pkg/promql"
+)
+
+func withThanosQueryInstantQueryGroup(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Instant Query",
+		panelgroup.PanelsPerLine(3),
+		panels.InstantQueryRequestRate(datasource, labelMatcher),
+		panels.InstantQueryRequestErrors(datasource, labelMatcher),
+		panels.InstantQueryRequestDurations(datasource, labelMatcher),
+	)
+}
+
+func withThanosQueryRangeQueryGroup(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Range Query",
+		panelgroup.PanelsPerLine(3),
+		panels.RangeQueryRequestRate(datasource, labelMatcher),
+		panels.RangeQueryRequestErrors(datasource, labelMatcher),
+		panels.RangeQueryRequestDurations(datasource, labelMatcher),
+	)
+}
+
+func withThanosQueryConcurrencyGroup(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Query Available Concurrency",
+		panelgroup.PanelsPerLine(1),
+		panels.QueryConcurrency(datasource, labelMatcher),
+	)
+}
+
+func withThanosQueryDNSLookupGroup(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("DNS Lookups",
+		panelgroup.PanelsPerLine(2),
+		panels.DNSLookups(datasource, labelMatcher),
+		panels.DNSLookupsErrors(datasource, labelMatcher),
+	)
+}
+
+func BuildThanosQueryOverview(project string, datasource string, clusterLabelName string) (dashboard.Builder, error) {
+	clusterLabelMatcher := dashboards.GetClusterLabelMatcher(clusterLabelName)
+	return dashboard.New("thanos-query-overview",
+		dashboard.ProjectName(project),
+		dashboard.Name("Thanos / Query / Overview"),
```

Comment:
did you checked if Perses will handle properly / on the dashboard name? I don't really remember

Reply:
Yeah I checked, seems to work everywhere.
![Screenshot 2025-04-30 at 11 14 57](https://github.com/user-attachments/assets/6e3b7ed3-e09b-4a94-a3da-0068690dec74)
![Screenshot 2025-04-30 at 11 15 47](https://github.com/user-attachments/assets/96eb07dc-4722-4614-9c20-c622e51d1466)